client/reader/search-stream/index.jsx

```diff
@@ -63,18 +66,28 @@ const SearchCardAdapter = React.createClass( {
 		this.props.handleClick( this.props.post, {} );
 	},
 
+	onRefreshCardClick( post ) {
+		recordTrackForPost( 'calypso_reader_searchcard_clicked', this.props.post );
+
+		event.preventDefault();
```

Comment:
:D You're right! This is left over from the old click handler and should be removed.